Models/Entities/ParkedVehicle.cs

```diff
@@ -10,9 +11,12 @@ public class ParkedVehicle
         [Required(ErrorMessage = "Vehicle type is required.")]
         public VehicleType VehicleType { get; set; }
 
-        [Required]
-        [MaxLength(10)]
-        [RegularExpression("^[A-Z0-9]*$", ErrorMessage = "Registration number should be uppercase alphanumeric.")]
+        //[Required]
+        //[MaxLength(10)]
+        //[RegularExpression("^[A-Z0-9]*$", ErrorMessage = "Registration number should be uppercase alphanumeric.")]
+        [Display(Name = "Registration Number")]
+        [Required(ErrorMessage = "Please enter a registration number.")]
+        [RegularExpression(@"^[A-Z]{3}\d{3}$", ErrorMessage = "Please enter a valid registration number with 3 uppercase letters and 3 numbers.")]
```

Comment:
We shouldn't enforce the 3 letters 3 numbers, because some vehicles have Unique license plates. I understand where you're coming from but we shouldn't enforce that. I will adjust it to previous RegularExpression